reactor-core/src/test/java/reactor/util/ConsoleLoggerTest.java

```diff
@@ -88,6 +88,26 @@ public void traceNulls() {
 				.contains("param null is null");
 	}
 
+	@Test
+	public void traceDismissedInNonVerboseMode() {
+		PrintStream out = System.out;
+		Logger log = new Loggers.ConsoleLogger("test", false);
+		System.setOut(new PrintStream(outContent));
```

Comment:
`errContent` is not set but asserted

Reply:
indeed... actually setting the System printstreams is not even necessary and doesn't help the test, so I simplified it